test/fixtures/controllers/PromiseMiddlewareTestController.js

```diff
@@ -1,9 +1,11 @@
 module.exports = function (BaseController, Promise) {
-
   class PromiseMiddlewareTestController extends BaseController {
-
     configure(app) {
       super.configure(app);
+
+      app.set('view engine', 'ejs');
+      app.set('views', `${__dirname}/views`);
```

Comment:
This is not needed. In the test you mock and inject ejs, where ever it's failing should be depending on PromiseMiddlewareTestController

You will need code like this in near the existing injector:

```
   const ejsMock = {
  render: jest.fn()
};

    injector()
      .addDependency('ejs', ejsMock, true)
      .inject((TasksPrinter) => {
        this.TasksPrinter = TasksPrinter;
      });

```

Reply:
`app.set('view engine', ...);` is not needed if the view name includes the extension, but setting the views directory is still needed even when mocking the view engine, otherwise `express` throws `Internal Server Error`